dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -151,6 +150,26 @@ using parameters::ParametersSetCollection;
 
 static const char TIME_FILENAME[] = "time.bin";  ///< name of the file to dump time at the end of the simulation
 
+
+/**
```

Comment:
Why is this required?

Reply:
The key of unordered_map<std::pair<std::string, std::string>, boost::shared_ptr<curves::Curve> > is a std::pair<std::string, std::string> and the Hash is not already defined so I had to define it.

Reply:
This was already true earlier right?

Reply:
It seems we didn't have to define it when we use boost::unordered_map